python/ray/dashboard/client/src/pages/dashboard/tune/Tune.tsx

```diff
@@ -83,8 +113,82 @@ class Tune extends React.Component<
     });
   };
 
+  handleExperimentChange = (event: React.ChangeEvent<{ value: any }>) => {
+    this.setState({
+      experiment: event.target.value,
+    });
+  };
+
+  handleExperimentSubmit = async () => {
+    this.setState({ loading: true });
+    try {
+      await setTuneExperiment(this.state.experiment);
+      window.clearTimeout(this.timeout);
+      await this.refreshTuneInfo();
+      this.setState({ loading: false });
+    } catch (error) {
+      this.props.setError(error.toString());
+      this.setState({ loading: false });
+    }
+  };
+
+  experimentChoice = (prompt: boolean) => {
+    const { classes } = this.props;
+
+    const { loading } = this.state;
+    return (
+      <div>
+        <Typography className={classes.warning} color="textSecondary">
+          <WarningRoundedIcon className={classes.warningIcon} /> Note: This tab
+          is experimental.
+        </Typography>
+
+        {prompt && (
+          <Typography className={classes.heading} color="textPrimary">
+            You can use this tab to monitor Tune jobs, their statuses,
+            hyperparameters, and more. For more information, read the
+            documentation{" "}
+            <a href="https://docs.ray.io/en/latest/ray-dashboard.html#tune">
+              here
+            </a>
+            .
+          </Typography>
+        )}
+        <div>
+          <Typography className={classes.prompt} color="textSecondary">
+            {" "}
+            Enter Tune Log Directory Here:{" "}
+          </Typography>
+          <TextField
+            className={classes.input}
+            id="standard-basic"
+            value={this.state.experiment}
+            onChange={this.handleExperimentChange}
+          />
+          <Button
+            className={classes.submit}
+            variant="outlined"
+            onClick={this.handleExperimentSubmit}
+          >
+            <Typography color="textSecondary" className={classes.warning}>
+              {" "}
+              Submit{" "}
+            </Typography>
```

Comment:
you shouldn't need to use typography within a button. You can add color and classnames to the button component itself.

Also, spaces should only be used as separators _between_ words in a body of text. For spacing outside of that, use css layout properties instead. (padding or margins)